tasks/restund.yml

```diff
@@ -61,15 +53,13 @@
 - name: install restund unit file
   template:
     src:   templates/restund.service.j2
-    dest:  /lib/systemd/system/restund.service
+    dest:  /etc/systemd/system/restund.service
     mode:  0644
     owner: root
     group: root
   notify:
     - systemctl daemon-reload
-    # - restart restund # NOTE: Restarting restund should be done _manually_
-    - rkt gc
-    - rkt image gc
+    - restart restund # NOTE: Restarting restund should be done _manually_
```

Comment:
I uncommented this as it was useful for testing. But we should probably recomment it before merging